docs/source/getting-started/ui.rst

```diff
@@ -112,6 +114,30 @@ The output tab controls how the contours which make it through thesholding and f
 
 This tab also allows teams to perform crosshair calibration. Instead of offsetting values in code, teams can line up their robot perfectly by hand, click "calibrate A" and "calibrate B", and the crosshair will be set to the current position. If the robot needs to shoot gamepieces into a goal from different distances, teams can calibrate A at their closest scoring position and B at their furthest scoring location, and the crosshair will linearly interpolate between the two offsets based on distance (area) from the target.
 
+.. _learn-ui-stream:
+
+Stream
+~~~~~~~~
+
+The stream tab controls settings for the video stream of each camera. 
+
+.. note::
+	These settings are per pipeline and NOT per camera.
+
+	Just to clearify, this is an FPS limit of the video stream, NOT the processing frame rate which is usually way higher
+
+You can adjust the FPS, 30 is recommended for driving and lower frame rates can be better for an intake system for example.
```

Comment:
```suggestion
It is possible to adjust the FPS. 30 FPS minimum is recommended for driving, however, lower frame rates can work in places such as an intake.
```

Is there an image of this?

Reply:
Would be a good idea to add how NT tables update 100 times per second, so having an fps greater than 100 would be bottlenecked.